data/human/human missions.txt

```diff
@@ -4964,3 +4964,77 @@ mission "Riley"
 			label end
 			`	You say your goodbyes, leaving to your ship and returning to the stars.`
 				decline
+
+mission "A dead man's path I"
+	name "Siblings to New China"
+	description "Bring these siblings to New China to hear their father's last will."
+	passengers 2
+	source
+		near "Merak" 3 6
+	destination
+		planet "New China"
+	to offer
+		random < 20
+	on offer
+		conversation
+			`As you walk through the spaceport, a young woman and man, who are both dressed very formally, approach you. The woman begins, "Hello, are you a spaceship captain?" You answer, "Indeed, I am." Her face brightens up with a smile, and she continues, "That is good to hear, my brother and I were searching for someone who could give us a ride to New China. We received notice that our father's last will will be published there. Do you think you could do this? We will pay for your service, of course."`
+			choice
+				"Of course I can."
+					goto "Agree"
+				"I am sorry, but I have more urgent things to do."
+			`    Her smile fades away as you say that, but they nod and turn away.`
+				decline
+			label "Agree"
+			`    "Fantastic," she says, "my name is Sophie. This is my younger brother," she points at the man to her side. "Hi, I am Liam," he says, "we have been looking for a ride for some time now, we already started to worry we wouldn't get there in time," he tells you. You nod in agreement. "A lot of captains are very busy these times, you are lucky that I have some time. My name is <first> <last>." They nod in agreement and then you lead them to your spaceship.`
+				accept
+	on complete
+		conversation
+			`When you arrive at New China's spaceport, the siblings head off to the place where their fathers last will will be read out. Liam turns to you. "Captain <last>, I have transferred the payment to your account, would it be okay for you to wait here until we come back? We will need a ride home." You agree, and go to the local bar to kill some time.`
+			`    After about two hours they come back, both seem somehow excited. You flag them down from your table, and they sit down. Sophie begins talking, "Captain <last>, it turns out we will need a bit more than just a ride home. If you want to hear us out, come here again in two hours because my brother and I need to buy some things."`
+		payment 50000
+
```

Comment:
you could also set a log upon completion of each part

Reply:
Noted